priv/rebar3_release/erlang_ls.config

```diff
@@ -0,0 +1,2 @@
+apps_dirs:
```

Comment:
This is all it's needed to support umbrella projects after this PR.

Reply:
How could this be used if the app dirs are directly in the project root?

Reply:
I think the following should work, this is the deafult value though (see [here](https://github.com/erlang-ls/erlang_ls/pull/308/files#diff-a40e5ac4b7520587b5f49f27230ed126R72)):
```
apps_dirs:
  - "."
```

Reply:
Yes, what @jfacorro said :)